### PR TITLE
Feature/1492/go utils refactoring

### PR DESCRIPTION
**Pull request by bacherfl**

### CODE REVIEW

File: api/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Use the offical Golang image to create a build artifact.
 # This is based on Debian and sets the GOPATH to /go.
-FROM golang:1.12.13-alpine as builder
+FROM golang:1.13-alpine as builder
```
Comment (christian-kreuzberger-dtx) [inline api/Dockerfile:3]: Please be specific in the golang version, as Dynatrace oneagent does not support newer versions yet.
Screenshot from 2020-04-03 12-22-36
I think 1.13.7 should be fine.

File: helm-service/controller/configuration_changer.go [hunk 2]
```diff
@@ -56,8 +56,13 @@ func (c *ConfigurationChanger) ChangeAndApplyConfiguration(ce cloudevents.Event,
 		return err
 	}
 
+	keptnHandler, err := keptnevents.NewKeptn(&ce, keptnevents.KeptnOpts{})
+	if err != nil {
+		c.logger.Error("Could not initialiue keptn handler: " + err.Error())
```
Comment (christian-kreuzberger-dtx) [inline helm-service/controller/configuration_changer.go:61]: Typo *initialiue*

**Comment by christian-kreuzberger-dtx** [comment]
Btw, when we merge this, we need to make sure that this is squash merged

File: api/Dockerfile (hunk 1 above)
Comment (bacherfl) [reply api/Dockerfile]: done

File: helm-service/controller/configuration_changer.go (hunk 2 above)
Comment (bacherfl) [reply helm-service/controller/configuration_changer.go]: done

**Pull request merged**